queue.tf

```diff
@@ -28,3 +28,26 @@ resource "azurerm_key_vault_secret" "notifications_queue_listen_conn_str" {
   name         = "notifications-queue-listen-connection-string"
   value        = "${module.notifications-queue.primary_listen_connection_string}"
 }
+
+module "notifications-staging-queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=master"
+  name                = "notifications-staging"
+  namespace_name      = "${module.queue-namespace.name}"
+  resource_group_name = "${azurerm_resource_group.reform_scan_rg.name}"
+  lock_duration       = "PT5M"
+
+  duplicate_detection_history_time_window = "PT15M"
+  create_queue       = "${var.create_staging_queue}"
+}
+
+resource "azurerm_key_vault_secret" "notifications_staging_queue_send_conn_str" {
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}"
+  name         = "notifications-staging-queue-send-connection-string"
+  value        = "${var.create_staging_queue == "true" ? module.notifications-staging-queue.primary_send_connection_string : "" }"
```

Comment:
I think "" as a key vault secret is not accepted.  Could you please try in portal/cli to create a secret with "" . I am sure this will fail. It will need some value may be space " " or "dummy_value" ?

Reply:
A value that says it's a dummy value is actually better, regardless of whether space works